install/kubernetes/cilium/templates/hubble-ca-configmap.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if and (not .Values.preflight.enabled) .Values.agent .Values.hubble.enabled .Values.hubble.tls.enabled (hasKey .Values.hubble "listenAddress") }}
+{{- if not (hasKey .Values "quickInstall") }}
```

Comment:
using hasKey means that we potentially ignore the actual value (true or false), it might lead to some confusion. What do think about setting default value of this attribute in values.yaml as `false`, and update test statement accordindly ?

Reply:
For context, the conclusion of an earlier discussion around its presence in values.yaml file was to avoid having it there and instead use this approach. Seems reasonable to also check for the content of the Value though.

https://github.com/cilium/cilium/pull/13607#discussion_r506674867